protocol/daemons/pricefeed/client/constants/testdata/usdt_exchange_config.json

```diff
@@ -40,9 +32,7 @@
     },
     {
       "exchangeName": "Okx",
-      "ticker": "BTC-USDT",
-      "adjustByMarket": "BTC-USD",
-      "invert": true
+      "ticker": "USDT-USDC"
```

Comment:
Is the ticker `USDT-USDC` ? Can we test this? Based on this https://www.okx.com/trade-spot/usdc-usdt it looks like it might be `USDC-USDT` same as binance

Reply:
I checked, and Okx has a USDT-USDC ticker. I filled this in based on the sheet row to update Okx USDT config to fetch USDT in USDC quote currency. I can switch to USDC-USDT and invert the market response if you prefer.